Exercise_1/day1/Program11.java

```diff
@@ -0,0 +1,25 @@
+package day1;
+
+public class Program11 {
+	public static void main(String[] args) {
+
+		  int n = 3;
+		  System.out.println("Pattern using Two Dimensional Matrix :");
+		  print(n);
+
+	  }
+	  public static void print(int n)
```

Comment:
ensure proper intendations for readability

Reply:
noted